gcc/rust/ast/rust-expr.h

```diff
@@ -178,6 +180,12 @@ class MetaItemPathLit : public MetaItem
     return path.as_string () + " = " + lit.as_string ();
   }
 
+  // There are two Locations in MetaItemPathLit (path and lit_expr),
+  //  we have no idea use which of them, just simply return UNKNOWN_LOCATION
+  //  now.
+  // Maybe we will figure out when we really need the location in the future.
+  Location get_locus () const override { return Location (UNKNOWN_LOCATION); }
```

Comment:
I think this should return the "earlier" of the two locations. In the case of `MetaItemPathLit`, which is of the form `path = lit`, I think this should return the path's location.
```suggestion
  Location get_locus () const override { return path.get_locus (); }
```